MongoBundle/Resources/config/transformer.yml

```diff
@@ -1,2 +1,7 @@
 parameters:
     open_orchestra.transformer.boolean_to_bdd.class: OpenOrchestra\Transformer\ConditionFromBooleanToMongoTransformer
+services:
+    open_orchestra.transformer.boolean_to_bdd:
+        class: %open_orchestra.transformer.boolean_to_bdd.class%
+        arguments:
+            - ''
```

Comment:
Parameter was added to constructor in an abstract class in order to put a type on the boolean-mongo transformer. This transformer could not be used as service because of this parameter and datatransformer process during which parameter could change. For internal link, I use the transformer in an other context in which i can set the parameter and use immediately the transformer avoiding parameter changes. and finally use it as service. So I keep the mandatory parameter in constructor when i can't use as service and set it to '' when i can use as service. @itkg-ngilain